src/Components/Details/Details.css

```diff
@@ -60,4 +61,16 @@ button:hover {
 
 .details__each-genre {
   padding-left: .75rem;
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 800px) {
+  .details__movie-title {
+    font-size: 40pt;
+ }
+}
+
+@media screen and (max-width: 350px) {
+  .details__movie-title {
+    font-size: 30pt;
+  }
+}
```

Comment:
Thanks for adding this in - like the extra shrink on the title text for those users still viewing on their iPhone5!